### DISCUSSION TITLE
Text to Speech Issues

### DISCUSSION THREAD

**Discussion by cdasso** [FrSkyRC/ETHOS-Feedback-Community#4657] [Q&A] [answered]
So I got a x20RS a week ago. It’s great but one of the things I was looking forward to was the text to speech function. I haven’t found anything been trying to use it but it’s a train wreck for me. The biggest issue is that it is spelling out very simple words like “hold” or “cut” instead of speaking them. Basically useless. Is there a trick that I am missing?

**Comment by msromike** [comment, marked as answer]
Lower case VS upper case.
